README.md

```diff
@@ -191,6 +191,10 @@ connection = Excon.new('http://geemus.com/', :connect_timeout => 360)
 
 # opt-out of nonblocking operations for performance and/or as a workaround
 connection = Excon.new('http://geemus.com/', :nonblock => false)
+
+# set shorter dns_timeouts for resolving addresses (default is 60 seconds)
```

Comment:
Looking at the resolv code, I think the default is actually a bit more complicated than this (at least in so far as I understand it). It appears to be an array of values that looks like `[ 5, second = 5 * 2 / nameserver_count, 2 * second, 4 * second ]`, see: https://github.com/ruby/ruby/blob/master/lib/resolv.rb#L342C7-L342C71. It does appear to allow for singular values, but it might be good to explain that it can be singular or an array here (and what to expect from each).

Reply:
I've just added some additional info.
Do you think it's enough?

Reply:
Yeah, I think that looks good. It gives people an overview and a place to look for details. If we got much more into the details here, I think there would be too much risk that it would fall out of date with Resolv, so it seems better to give a description and a reference directly instead.